docs/data/DATA.md

```diff
@@ -1,12 +1,12 @@
 # Data
 
-Data is currently generated from a Wikidata JSON dump loaded into Hadoop on the Wikimedia Foundation Analaytics cluster. See DATA-NEW.md
+Data is currently generated from a Wikidata JSON dump loaded into Hadoop on the Wikimedia Foundation Analaytics cluster, see [DATA-NEW.md](DATA-NEW.md).
 
-Old data can also be gathered from the old wikidata-analysis project, See DATA-OLD.md
+Old data can also be gathered from the old [wikidata-analytics]([url](https://github.com/vrandezo/wikidata-analytics)) project, see [DATA-OLD.md](DATA-OLD.md).
```

Comment:
Is this the correct project?

Reply:
I think so

Reply:
Wait, no, not in the cpontext of this sentence.
This github repo is the very very very old origins of this whole code and idea.
wikidata-analytics is the wmflabs project / tool
I dont appear to be able to find a link to it currently.

Reply:
Okay, I've removed that link then